src/lib/network/edit_nic_name.rb

```diff
@@ -151,5 +162,36 @@ def CheckUdevNicName(name)
 
       true
     end
+
+    # When an interface name has changed, it returns whether the user wants to
+    # update the interface name in the related routes or not.
+    #
+    # return [Boolean] whether the routes have to be updated or not
+    def update_routes?(previous_name)
+      return false unless Routing.device_routes?(previous_name)
+
+      Popup.YesNoHeadline(
+        Label.WarningMsg,
+        # TRANSLATORS: Ask for fixing a possible conflict after renaming
+        # an interface, %s are the previous and current interface names
+        format(_("The interface %s has been renamed to %s. There are \n" \
+                  "some routes that still use the previous name.\n\n" \
+                  "Would you like to update them now?\n"),
+          "'#{previous_name}'",
+          "'#{LanItems.current_name}'")
+      )
+    end
+
+    # It modifies the interface name with the new one of all the routes
+    # that belongs to the current renamed {LanItem}
+    def update_routes!(previous_name)
+      Routing.device_routes(previous_name).each do |route|
+        route["device"] = LanItems.current_name
+      end
+    end
+
+    def update_routing_devices!
+      Routing.SetDevices(LanItems.current_device_names)
+    end
```

Comment:
again?

Reply:
It is a convenience method for update the routing devices, preferred to have it duplicated than do a wrong abstraction.